src/sentry/static/sentry/app/views/releasesV2/detail/overview/projectReleaseDetails.tsx

```diff
@@ -29,6 +29,15 @@ const ProjectReleaseDetails = ({release}: Props) => {
             </TagValue>
           </StyledTr>
 
+          {lastDeploy?.dateFinished && (
+            <StyledTr>
+              <TagKey>{t('Last Deploy')}</TagKey>
+              <TagValue>
+                <DateTime date={lastDeploy.dateFinished} seconds={false} />
```

Comment:
Should the last deploy tell you which environment it was for?

Reply:
There's an open issue in asana to redesign the way we show deploy badges in the header of this page - we would like to show dates there.

I added the environment tooltip to last deploy in the meantime 👍